build.gradle

```diff
@@ -56,7 +56,7 @@ repositories {
 }
 
 ext {
-    opensearch_version = System.getProperty("opensearch.version", "1.3.0-SNAPSHOT")
+    opensearch_version = System.getProperty("opensearch.version", "2.0.0-SNAPSHOT")
```

Comment:
Can we make this version change separate from the removal of the transport client references?  This change should be able to merge no matter the state of OpenSearch

Reply:
Yes, this is a draft PR for experiment to identify how far security plugin is from supporting OpenSearch 2.0.0. This single line version bump can be in a separate PRs once all required changes for OpenSearch 2.0.0 support are ready.